src/lib.rs

```diff
@@ -98,14 +98,16 @@ mod errors;
 pub use errors::{DeviceTypeError, EepromStringsError, EepromValueError, FtStatus, TimeoutError};
 
 mod mpsse;
+use libftd2xx_ffi::FT_EEPROM_X_SERIES;
```

Comment:
There's an import for all `libftd2xx_ffi` types below this, lump it in with those.